src/Utilities/FlowAnalysis/Framework/DataFlow/AbstractAnalysisData.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.FlowAnalysis.DataFlow
+{
+    internal abstract class AbstractAnalysisData : IDisposable
+    {
+        public bool Disposed { get; private set; }
```

Comment:
nit: IsDisposed?  Or is Disposed more conventional?

Reply:
Sure, can change.